Signal/src/ViewControllers/AppSettings/OWSLinkedDevicesTableViewController.m

```diff
@@ -48,6 +48,7 @@ - (void)viewDidLoad
     self.isExpectingMoreDevices = NO;
     self.tableView.rowHeight = UITableViewAutomaticDimension;
     self.tableView.estimatedRowHeight = 60;
+    self.tableView.separatorColor = Theme.hairlineColor;
```

Comment:
Can we instead use/adjust the existing `Theme.cellSeparatorColor` ?

Reply:
Done.